app-editors/okteta/okteta-0.26.13.ebuild

```
@@ -54,9 +54,16 @@ DEPEND="
 RDEPEND="${DEPEND}"
 
 src_configure() {
+	if ! use handbook; then
+		cmake_comment_add_subdirectory doc
+		sed -e 's/DocTools//;s/kdoctools_install(po)//' -i CMakeLists.txt
```

External commands which may [fail](https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html) should always have a `|| die`:
```suggestion
		sed -e 's/DocTools//;s/kdoctools_install(po)//' -i CMakeLists.txt || die
```